thirdparty/patchmatchnet/models/patchmatch.py

```diff
@@ -140,10 +140,9 @@ def forward(
             height: depth map height,
             width: depth map width,
             depth_sample: sample depth map, in shape of [batch, num_depth, height, width],
-            grid: 2D grid for bilinear gridding, in shape of [batch, len(original_offset)*H, W, 2],
-                    len(original_offset) generally equals to 9, suggesting the 3x3 neighbor grids.
-            depth_min: minimum virtual depth,
-            depth_max: maximum virtual depth,
+            grid: 2D grid for bilinear gridding, in shape of [batch, neighbors*H, W, 2]

```

Comment:
@codyly shape is usually a tuple, so i think it's clear to say that here, as
```
grid: 2D grid for bilinear gridding, in shape of (B, neighbors*H, W, 2), where B is the batch size
epth_min: minimum virtual depth, in shape of (B,)
```
I think B is clearer for batch size